examples/tenant.yaml

```
@@ -181,6 +181,13 @@ spec:
   #   value: "off" # to turn-off browser
   # - name: MINIO_STORAGE_CLASS_STANDARD
   #   value: "EC:2"
+  #   ## For secure env vars like passwords, create a Kubernetes secret and refer the secret under the `valueFrom` field
```

@nitisht am assuming that `secretKeyRef` here is the name of the secret, and `name` maps to the field in the opaque secret? If so, suggest the following

```suggestion
  #   ## For secure env vars like passwords, create an opaque Kubernetes secret and specify the secret name in the `valueFrom` field. The `valueFrom:<secretName>` object must contain the following fields:
  #   ## - `name` - the secret field from which MinIO extracts the password
  #   ## - `key` - the environment variable to which MinIO associates the secret
```

---

made a minor change to ensure correct meaning @ravindk89 PTAL